zipkin/include/zipkin/tracer.h

```diff
@@ -9,6 +9,7 @@ namespace zipkin {
 const SteadyClock::duration DEFAULT_REPORTING_PERIOD =
     std::chrono::milliseconds{500};
 const size_t DEFAULT_SPAN_BUFFER_SIZE = 1000;
+const long DEFAULT_TRANSPORT_TIMEOUT = 0;
```

Comment:
Please use `std::chrono::milliseconds` for this instead of a plain integral type.

Reply:
@rnburn updated